scripts/version.js

```diff
@@ -7,4 +7,7 @@ import * as paths from '../lib/paths.js'
 const pkg = JSON.parse(await fs.readFile(paths.packageJSON, 'utf8'))
 pkg.sentryEnvironment = 'production'
 await fs.writeFile(paths.packageJSON, JSON.stringify(pkg, null, 2) + '\n')
-await execa('git', ['add', 'package.json'])
+await execa('git', [
+  'add',
+  'package.json',
+])
```

Comment:
Reading this PR more and more, I think I would like to remove the multi line array plugin. The desire to keep diffs clean is imposing a too inconvenient style